src/features/modals/CreateToken/CreateToken.tsx

```diff
@@ -0,0 +1,106 @@
+//- React Imports
+import React, { FC, useState } from 'react';
+
+//- Style Imports
+import styles from './CreateToken.module.scss';
+
+//- Component Imports
+import { Wizard } from '@zero-tech/zui/src/components/Wizard/Wizard';
+import { CreateTokenBody } from './CreateTokenBody/CreateTokenBody';
+import { CreateTokenHeader } from './CreateTokenHeader/CreateTokenHeader';
+
+//- Type Imports
+import { Step } from '@zero-tech/zui/src/components/StepBar/StepBar.types';
+import { DetailsFormSubmit, TokenomicsFormSubmit } from './CreateToken.types';
+import { MediaType } from '@zero-tech/zui/src/components/MediaInput';
+
+const steps: Step[] = [
+	{
+		id: 'details',
+		title: '1/3 Details',
```

Comment:
Hmm I think the `1/3` is something we should probably handle in zUI. Can you change this to remove the `x/y` numbers and I'll add to zUI?